mk/main.mk

```diff
@@ -20,7 +20,9 @@ CFG_RELEASE_NUM=1.1.0
 # versions (section 9)
 CFG_PRERELEASE_VERSION=.1
 
-CFG_FILENAME_EXTRA=4e7c5e5c
+# Append a version-dependent hash to each library, so we can install different
+# versions in the same place
+CFG_FILENAME_EXTRA=$(shell echo -n $(CFG_RELEASE) | $(CFG_HASH_COMMAND))
```

Comment:
Good point. I think I'll use `printf '%s'` instead, which has more consistent behavior than `echo`.

The configure script depends on `printf` already, so it should be okay to use it.